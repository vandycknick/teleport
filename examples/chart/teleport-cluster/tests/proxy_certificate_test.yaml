suite: Proxy Certificate
templates:
  - proxy/certificate.yaml
tests:
  - it: should request a certificate for cluster name when cert-manager is enabled (cert-manager.yaml)
    values:
      - ../.lint/cert-manager.yaml
    asserts:
      - hasDocuments:
          count: 1
      - isKind:
          of: Certificate
      - matchSnapshot:
          path: spec.dnsNames
      - matchSnapshot:
          path: spec.issuerRef

  - it: should request a certificate for cluster name when cert-manager is enabled (cert-secret.yaml)
    values:
      - ../.lint/cert-secret.yaml
    asserts:
      - hasDocuments:
          count: 1
      - isKind:
          of: Certificate
      - matchSnapshot:
          path: spec.dnsNames
      - matchSnapshot:
          path: spec.issuerRef
