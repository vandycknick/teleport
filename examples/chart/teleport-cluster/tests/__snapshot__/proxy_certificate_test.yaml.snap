should request a certificate for cluster name when cert-manager is enabled (cert-manager.yaml):
  1: |
    - test-cluster
    - '*.test-cluster'
  2: |
    group: custom.cert-manager.io
    kind: CustomClusterIssuer
    name: custom
should request a certificate for cluster name when cert-manager is enabled (cert-secret.yaml):
  1: |
    - test-cluster
    - '*.test-cluster'
  2: |
    group: cert-manager.io
    kind: Issuer
    name: letsencrypt
